pkgs/development/tools/analysis/banana-vera/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python310
+, boost
+, tcl
+}:
+
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
We need to add this for `src.rev`.